app/views/tag_wranglers/show.html.erb

```diff
@@ -4,9 +4,14 @@
     <h2 class="heading"><%= ts("%{wrangler_login}'s Wrangling Page", :wrangler_login => @wrangler.login) %></h2>
     <p class="icon"><%= icon_display(@wrangler) %></p>
     <!--/descriptions-->
+    <!--subnav-->
+    <% if policy(:tag_wrangler).report_csv? %>
+      <ul class="navigation actions" role="navigation">
+        <li><%= link_to t(".tags_wrangled_csv"), report_csv_tag_wrangler_path %></li> <%# TODO #%>
```

Comment:
TODO?

Also, out of curiosity, by which Rails magic does `report_csv_tag_wrangler_path` manage to determine the id without having `@wrangler` as parameter?

Reply:
My bad, that's left over from my initial testing :smile: 

I'm actually not 100% sure about that, magic but I suspect it has something to do with [this bit in ActionDispatch::Routing::RouteSet](https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/routing/route_set.rb#L324).